src/model/BoxPlotColumn.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Created by bikramkawan on 24/11/2016.
+ */
+import * as d3 from 'd3';
+import ValueColumn from './ValueColumn';
+import Column from './Column';
+import {IDataStat} from './MultiValueColumn';
+
+
+function numSort(a, b) {
+
+  return a - b;
+}
+
+export enum Sort {
+
+  min, max, median, q1, q3, mean
+}
+
+
+export interface IBoxPlotColumn {
+  getBoxPlotData(row: any, index: number, scale: any): IBoxPlotData;
+  getDataInfo(): IDataStat;
+  getUserSortBy(): string;
+
+}
+
+interface IBoxPlotDataStat extends IDataStat {
```

Comment:
Yes. This was just extended for testing purpose in targid case where we need to set the domain. So I have to overwrite the IDataStat attributes.